pkg/controllers/v1alpha1/dataload/implement.go

```diff
@@ -209,7 +210,12 @@ func (r *DataLoadReconcilerImplement) reconcilePendingDataLoad(ctx reconcileRequ
 	case common.ALLUXIO_RUNTIME:
 		podName := fmt.Sprintf("%s-master-0", targetDataset.Name)
 		containerName := "alluxio-master"
-		fileUtils := operations.NewAlluxioFileUtils(podName, containerName, targetDataset.Namespace, ctx.Log)
+		fileUtils := alluxioOperations.NewAlluxioFileUtils(podName, containerName, targetDataset.Namespace, ctx.Log)
+		ready = fileUtils.Ready()
+	case common.JINDO_RUNTIME:
+		podName := fmt.Sprintf("%s-jindofs-master-0", targetDataset.Name)
```

Comment:
suggest put `%s-jindofs-master-0` in a unified place

Reply:
thx,same situation as alluxio , will resolve next pr